### ISSUE TITLE
Return back a logarithmic scale

### ISSUE THREAD

**Issue by ximik3** [CSSEGISandData/COVID-19#1308]
Since the cases of virus spreading have an exponential not a linear behaviour logarithmic scale introduced recently was the best visual representation of the situation. For some reason it was removed today.
Please return it back.
Thanks

**Comment by nguyandy** [comment]
My dashboard has a feature that lets you view the numbers on a logarithmic scale.
You can compare the rate of growth for countries you select as well.

https://covid19.nguy.dev

Hope that helps.

**Comment by Chris-JAC** [comment]
Agreed.  The logarithmic scale is very helpful, especially for determining when there is a change.
(And thanks for all the work you are doing on this).
